toc/rfc/rfc-000-template.md

```diff
@@ -1,4 +1,13 @@
-# My RFC Title
+# Meta
+[meta]: #meta
+- Name: (fill in the feature name: My Feature)
+- Start Date: (fill in today's date: YYYY-MM-DD)
+- Author(s): (Github usernames)
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CFC Pull Request: (leave blank)
+- CFC Issue: (leave blank)
```

Comment:
What does `CFC` stand for?

Reply:
I was thinking Cloud Foundry Community

Reply:
And the metadata fields would be used for linking implementation issues and PRs, right?

Reply:
Yep. There may be the desire to add some automation 

> > > #173 mentions WG specific RFCs
> > > > Provide GitHub Project per Working Group for large technical proposals/new feature RFCs).
> > > 
> > > 
> > > I understood this RFC process for common processes across WGs and crosscutting technical proposals. This should be clarified/mentioned in my opinion and maybe also add a link where to find the WG specific proposals.
> > 
> > 
> > I'm a little torn on this, should we have all the RFC's in this repo, possibly under a folder per WG structure to give one central place to make discovery and reuse easier? Or should we go with each WG having a separate set of RFCs, which may fit easier into their workflows and create more focussed discussion on the PRs?
> 
> We definitely need to leave room for working groups like Paketo to opt-out of centrally maintained RFCs. Other than that, we could use labels or folders or both and have focussed discussions independent from where those RFCs are located.
> 
> Having said that, I have no strong opinion on RFC placement.

How about a Folder with a link to where the RFCs could be found if they are not in this repo?

Reply:
I don't quite understand: how are these links to an issue and/or a PR on the community(?) repo different from the RFC PR on this repo? I don't see any discussion of those links in the RFC process, either.